bin/streamr-stream-grant-permission.ts

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node -r ts-node/register
+import { Command } from 'commander';
+import {
+    envOptions,
+    authOptions,
+    formStreamrOptionsWithEnv
+} from './common'
+import pkg from '../package.json'
+import { AnonymousStreamPermisson, StreamOperation, StreamrClient, UserStreamPermission } from 'streamr-client';
+import EasyTable from 'easy-table'
+
+const PUBLIC_PERMISSION_ID = 'public'
+
+const OPERATIONS: Record<string,StreamOperation> = {
+    'get': StreamOperation.STREAM_GET,
+    'edit': StreamOperation.STREAM_EDIT,
+    'delete': StreamOperation.STREAM_DELETE,
+    'publish': StreamOperation.STREAM_PUBLISH,
+    'subscribe': StreamOperation.STREAM_SUBSCRIBE,
+    'share': StreamOperation.STREAM_SHARE
+}
+
+const getOperation = (id: string) => {
+    const operation = OPERATIONS[id]
+    if (operation === undefined) {
+        console.error(`error: invalid operation: ${id}`)
```

Comment:
Wondering if we should enforce this here? If any new permissions get introduced in Core API / Client, we also need to remember to update the the `OPERATIONS` list in this file. But we could just let the back-end do the validation.

Reply:
Dropped this client-side validation